quantities/factories/IFactory.cs

```diff
@@ -0,0 +1,12 @@
+namespace Quantities.Factories;
+
+public interface IFactory { /* marker interface */ }
+
+public interface IFactory<in TFactory, out TTo, out TCreate> : IFactory
```

Comment:
This is a "pseudo" interface that "defines" how to "spell" the create (`Of`) and convert (`To`) members.

What `To` and `Of` actually return is implementation specific.

Reply:
Note also that most enabling types are in the folder "factories".